src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/ChangeRequestsTab.tsx

```diff
@@ -0,0 +1,129 @@
+        .sort((a, b) => b.dateSubmitted.getTime() - a.dateSubmitted.getTime())
+    : [];
+
+  const crApproved = changeRequests
+    ? changeRequests
+        .filter(
+          (cr: ChangeRequest) =>
+            cr.dateReviewed &&
+            cr.accepted &&
+            currentDate.getTime() - cr.dateReviewed.getTime() <= 1000 * 60 * 60 * 24 * 5 &&
+            projectWbsPipe(cr.wbsNum) === projectWbsPipe(workPackage.wbsNum)
+        )
+        .sort((a, b) => (a.dateReviewed && b.dateReviewed ? b.dateReviewed.getTime() - a.dateReviewed.getTime() : 0))
+    : [];
+  const displayCRCards = (crList: ChangeRequest[]) => (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'row',
+        flexWrap: 'wrap',
+        overflow: 'auto',
+        justifyContent: 'flex-start',
+        '&::-webkit-scrollbar': {
+          height: '20px'
+        },
+        '&::-webkit-scrollbar-track': {
+          backgroundColor: 'transparent'
+        },
+        '&::-webkit-scrollbar-thumb': {
+          backgroundColor: theme.palette.divider,
+          borderRadius: '20px',
+          border: '6px solid transparent',
+          backgroundClip: 'content-box'
+        }
+      }}
+    >
+      {crList.map((cr: ChangeRequest) => (
+        <ChangeRequestDetailCard changeRequest={cr}></ChangeRequestDetailCard>
+      ))}
+    </Box>
+  );
+
+  const renderChangeRequests = (title: string, crList: ChangeRequest[], emptyMessage: string) => {
+    return (
+      <>
+        <Typography variant="h5" gutterBottom>
+          {title}
+        </Typography>
+        {crList.length > 0 ? (
+          <Grid container>{displayCRCards(crList)}</Grid>
+        ) : (
+          <Typography gutterBottom>{emptyMessage}</Typography>
+        )}
+      </>
+    );
+  };
+  return (
+    <>
+      <Grid
+        sx={{
+          mb: 1,
+          mt: 3
+        }}
+      >
+        {renderChangeRequests('Un-reviewed Change Requests', crUnreviewed, 'No un-reviewed change requests')}
+        {renderChangeRequests('Recently Approved Change Requests', crApproved, 'No recently approved change requests')}
+      </Grid>
+
+      <Typography
+        variant="h5"
+        sx={{
+          mb: 1,
+          mt: 3
+        }}
+      >
+        Blocked By
+      </Typography>
+
+      <Stack direction="row" alignItems="center" divider={<Divider orientation="vertical" flexItem />} spacing={2}>
+        {dependencies.length === 0 ? (
+          <Typography>No Blockers</Typography>
+        ) : (
+          dependencies.map((dep, idx) => (
+            <Link
+              key={`${dep.name}-${idx}`}
+              component={RouterLink}
+              to={routes.PROJECTS + `/${wbsPipe(dep.wbsNum)}`}
+              fontWeight="bold"
+            >
+              {`${wbsPipe(dep.wbsNum)} - ${dep.name}`}
+            </Link>
+          ))
+        )}
+      </Stack>
+    </>
+  );
+};
+
```

Comment:
You can remove this whole blocked by section since its already on the overview tab